dev/com.ibm.ws.anno_fat/fat/src/com/ibm/ws/tests/anno/FATSuite.java

```diff
@@ -67,8 +71,12 @@
     MetadataIncompleteTest.class,
     FragmentOrderTest.class,
     LooseConfigTest.class,
-    AppReplaceTest.class
+
```

Comment:
```suggestion
AppReplaceTest.class
```

On merge, you removed this test from the list and it shouldn't be removed.